src/yarpWholeBodyStates.cpp

```diff
@@ -744,7 +751,30 @@ bool yarpWholeBodyEstimator::threadInit()
     left_sole_local_id = wbi::ID(LEFT_LEG,8);
     right_sole_local_id = wbi::ID(RIGHT_LEG,8);
     */
-
+    
+    /*    Eigen::Map<Eigen::VectorXd> dqjVect(dqj,dof);
+    Eigen::Matrix<double,6,Eigen::Dynamic,Eigen::RowMajor> complete_jacobian(6,dof+6), joint_jacobian(6,dof),floatingBase_jacobian(6,6);
+    Eigen::Matrix<double,6,Eigen::Dynamic,Eigen::RowMajor> tempMatForComputation(6,dof);//, minusTemp(6,dof);
+    Eigen::VectorXd dvbVect(6);
+    Eigen::Map<Eigen::VectorXd> yarpWrapper(estimates.lastBaseVel.data(), estimates.lastBaseVel.size());
+  */  
+/*
+    int dof = estimates.lastQ.length();
+    
+    complete_jacobian.resize(6,dof+6);
+    joint_jacobian.resize(6,dof);
+    floatingBase_jacobian(6,6);
+    tempMatForComputation.resize(6,dof);
+    dvbVect.resize(6);
+    new(&rotationalVelocityWrapper) Eigen::Map<Eigen::VectorXd>(estimates.lastBaseVel.data(), estimates.lastBaseVel.size());
+    //Eigen::Map<Eigen::VectorXd> yarpWrapper;//(estimates.lastBaseVel.data(), estimates.lastBaseVel.size());
+    //new (&v) Map<RowVectorXi>(data+4,5);
+    /*
+    std::cout<<" CompleteJacobian size : "<<complete_jacobian.rows()<<"X"<<complete_jacobian.cols()<<"\n";
+    std::cout<<" JointJacobian size : "<<joint_jacobian.rows()<<"X"<<joint_jacobian.cols()<<"\n";
+    std::cout<<" FloatingBaseJacobian size : "<<floatingBase_jacobian.rows()<<"X"<<floatingBase_jacobian.cols()<<"\n";
+    std::cout<<" TempMat size : "<<tempMatForComputation.rows()<<"X"<<tempMatForComputation.cols()<<"\n";
+    std::cout<<" rotationalVelocityWrapper size : "<<rotationalVelocityWrapper.rows()<<"\n\n";*/
```

Comment:
the commented bits might be useful in case of issues with other
yarp-compatible robot models which I havent got around to yet...


---
Naveen Kuppuswamy, PhD
Post-doctoral Fellow,
Cognitive Humanoids Lab,
Department of Robotics, Brain and Cognitive Sciences (RBCS),
Istituto Italiano di Tecnologia, Genova, Italy

On Fri, Feb 20, 2015 at 3:30 PM, Silvio Traversaro <notifications@github.com
> wrote:

> In src/yarpWholeBodyStates.cpp
> <https://github.com/robotology-playground/yarp-wholebodyinterface/pull/23#discussion_r25072305>
> :
>
> > +    int dof = estimates.lastQ.length();
> > +
> > +    complete_jacobian.resize(6,dof+6);
> > +    joint_jacobian.resize(6,dof);
> > +    floatingBase_jacobian(6,6);
> > +    tempMatForComputation.resize(6,dof);
> > +    dvbVect.resize(6);
> > +    new(&rotationalVelocityWrapper) Eigen::Map<Eigen::VectorXd>(estimates.lastBaseVel.data(), estimates.lastBaseVel.size());
> > +    //Eigen::Map<Eigen::VectorXd> yarpWrapper;//(estimates.lastBaseVel.data(), estimates.lastBaseVel.size());
> > +    //new (&v) Map<RowVectorXi>(data+4,5);
> > +    /*
> > +    std::cout<<" CompleteJacobian size : "<<complete_jacobian.rows()<<"X"<<complete_jacobian.cols()<<"\n";
> > +    std::cout<<" JointJacobian size : "<<joint_jacobian.rows()<<"X"<<joint_jacobian.cols()<<"\n";
> > +    std::cout<<" FloatingBaseJacobian size : "<<floatingBase_jacobian.rows()<<"X"<<floatingBase_jacobian.cols()<<"\n";
> > +    std::cout<<" TempMat size : "<<tempMatForComputation.rows()<<"X"<<tempMatForComputation.cols()<<"\n";
> > +    std::cout<<" rotationalVelocityWrapper size : "<<rotationalVelocityWrapper.rows()<<"\n\n";*/
>
> This is a bit block of code commented, I guess we can get rid of it.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/robotology-playground/yarp-wholebodyinterface/pull/23/files#r25072305>
> .
>